### DISCUSSION TITLE
using custom name instead of _count

### DISCUSSION THREAD

**Discussion by amirhhashemi** [prisma/prisma#8996] [Q&A] [answered]
Quick question. I have this query:
```ts
    const user = await this.prisma.user.findUnique({
      where : {id},
      select: {
        _count: {
          select: { followers: true, following: true },
        },
      },
    });
```
that returns:
```json
{
  "user": {
    "_count": {
      "followers": 0,
      "following": 0
    }
  }
}
```

what if I want to get this:
```json
{
  "user": {
    "followers_count": 0
    "following_count": 0
  }
}
```
**EDIT: I also asked this question in StackOverflow. consider answering there:**
https://stackoverflow.com/questions/68974802/how-to-use-a-custom-name-instead-of-count-in-prisma

**Comment by ryands17** [comment, marked as answer]
@ahhshm 👋 

This isn't possible at the moment. It would be great if you could create a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this.

**Reply by amirhhashemi** [reply]
Thanks for the reply.
I created a feature request here: https://github.com/prisma/prisma/issues/9004

**Comment by IRediTOTO** [comment]
I do think this will never do, want a custom var? You will lose the benefit of typescript. Btw object return above is fine, you really don't need custom it. If yes
```
const customData= {
  "user": {
    "followers_count": user._count.followers
    "following_count": user._count.following
  }
}
```

**Reply by amirhhashemi** [reply]
yes, the query result is fine. I just wanted to see if there is a better way or not. BTW thanks for the suggestion

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
